src/main/java/sirius/biz/jobs/batch/BatchProcessJobFactory.java

```diff
@@ -213,16 +216,36 @@ protected void executeTask(ProcessContext process) throws Exception {
     protected abstract BatchJob createJob(ProcessContext process) throws Exception;
 
     protected void logParameters(ProcessContext process) {
+        List<Parameter<?, ?>> parameters = getParameters();
+
         StringBuilder output = new StringBuilder();
         output.append("Parameter:\n\n");
```

Comment:
i18n

Reply:
couldn't we re-use the code and call it twice with a different heading and predicate (`Parameter::shouldBeSystemLogged` or `p -> !p.shouldBeSystemLogged()`)

Reply:
scratch that - add a getVisibility() +- parameter and simply pass in the desired visibility type (other normal or system)...